docker-compose.yml

```diff
@@ -154,6 +154,11 @@ services:
       dockerfile: ./apps/listing-service/Dockerfile
     image: nodejs-msa-listing:dev
     container_name: listing
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
```

Comment:
Should be Priorities 
- POSTGRES_HOST
- POSTGRES_PORT
- POSTGRES_USER
- POSTGRES_PASSWORD